### ISSUE TITLE
Further problem processing explicit s30 tendl photonuclear

### ISSUE THREAD

**Issue by jchsublet** [njoy/NJOY2016#204]
Although NJOY2016.64 cleared an important hurdle in processing specific explicit g-tendl evaluations, while testing it on many more targets, I came across another hurdle on a few: ***error in acephn***too many reactions in mtr list. In acephn mmax is set to 80 and the following example fails

[g-Zr092.txt](https://github.com/njoy/NJOY2016/files/6585758/g-Zr092.txt)
[inZr092.txt](https://github.com/njoy/NJOY2016/files/6585745/inZr092.txt)

I am baffled as if yes it appears that in the above case there are more than 80 open channels: 19 in mf3, 9x2 in mf10 and 142 in mf6/mt5, this also seem to be the case in say for example Zr091 counting 19 mf3, 7x2 mf10 and 141 in mf6/mt5 however this later one works with an output printed mtr=139 !!

[inZr091.txt](https://github.com/njoy/NJOY2016/files/6585956/inZr091.txt)
[g-Zr091.txt](https://github.com/njoy/NJOY2016/files/6585966/g-Zr091.txt)
[outZr091.txt](https://github.com/njoy/NJOY2016/files/6586047/outZr091.txt)
[Zr091chk.pdf](https://github.com/njoy/NJOY2016/files/6586053/Zr091chk.pdf)

**Comment by jchsublet** [comment]
To my suprise setting mmax=200 in fix/tendl-pn acepn.f90 correct the above error and all others error of the same type that occurred for 37 evaluations

**Comment by whaeck** [comment]
If it's that easy, I'll make the change then :-)

Discrete photons tend to get a different MT number associated to each one of them, so that might be it.

I'm currently looking at another issue (the MF34 issue in U235 and U238 that has been bugging me for a long time). Once I finish with that, I'll make this change - probably as NJOY2016.65. I'll add one of the nuclides you mention as a test case as well.

**Comment by whaeck** [comment]
I had a quick look at this. The issue is that NJOY2016 goes over the index in MF1 MT451 and looks at every single mf,mt entry with mf>3 as a "channel" that it will treat later (the mmax limit actually applies to the length of the MF1 MT451 index). This means that the covariance entries are counted here as well - which makes no sense.

I've increased the value of max to 200 as suggested, and I also made sure that covariance sections are ignored in the analysis of the index. See fix/tendl-pn for the updated code. Note: I will pull this into the develop branch since this is not a crucial fix (contrary to the previous one in which a corrupt ACE file was created).
